fury/tests/test_actors.py

```diff
@@ -1560,13 +1559,121 @@ def test_billboard_actor(interactive=False):
         centers, colors=colors, scales=scales, fs_impl=fake_sphere
     )
     scene.add(billboard_actor)
-    scene.add(actor.axes())
+
     if interactive:
         window.show(scene)
 
     arr = window.snapshot(scene)
     report = window.analyze_snapshot(arr, colors=colors)
     npt.assert_equal(report.objects, 8)
+    scene.clear()
+
+    centers = np.array([[0, 0, 0], [-15, 15, -5], [10, -10, 5],
+                        [-30, 30, -10], [20, -20, 10]])
+    colors = np.array([[1, 1, 0], [0, 0, 1], [1, 0, 1],
+                       [1, 0, 0], [0, 1, 0]])
+    scales = [3, 1, 2, 1, 1.5]
+
+    b_point = """
+        float len = length(point);
+        float radius = .2;
+        if(len > radius)
+            {fragOutput0 = vec4(vec3(0,0,0), 1);}
+        else
+            {fragOutput0 = vec4(color, 1);}
+        """
+
+    spherical_billboard = actor.billboard(centers, colors=colors,
+                                          scales=scales, bb_type='spherical',
+                                          fs_impl=b_point)
+
+    scene.add(spherical_billboard)
+    scene.add(actor.axes())
+    arr = window.snapshot(scene)
+    report = window.analyze_snapshot(arr, colors=255 * colors)
+    npt.assert_equal(report.colors_found, [True] * 5)
+
+    scene.pitch(87)
+    if interactive:
+        window.show(scene)
+
+    scene.reset_camera()
+    scene.reset_clipping_range()
+    arr = window.snapshot(scene, offscreen=True)
+    report = window.analyze_snapshot(arr, colors=255 * colors)
+    npt.assert_equal(report.colors_found, [True] * 5)
+
+    scene.pitch(-87)
+    scene.yaw(87)
+    if interactive:
+        window.show(scene)
+
+    scene.reset_camera()
+    scene.reset_clipping_range()
+    arr = window.snapshot(scene, offscreen=True)
+    report = window.analyze_snapshot(arr, colors=255 * colors)
+    npt.assert_equal(report.colors_found, [True] * 5)
+
```

Comment:
Huge test for doing the exact same things.   Can you just create a small loop? thanks